tests/translator/output/api_request_model.json

```diff
@@ -27,7 +18,16 @@
               }
             }
           ]
-        }
+        }, 
```

Comment:
Looks like IDE auto formatting. Can you revert these changes?

This really doesn't hurt anything, but we do use changes in "unrelated" outputs as an indication of possible regressions. Just makes sure we (as reviewers) are understanding the changes correctly and the PRs are organized to show all changes for the implementation.

Reply:
Removed it from the git, sorry, didn't realize it had changed it.